appveyor.yml

```diff
@@ -0,0 +1,49 @@
+environment:
+  # There is no need to run the build for all the Python version /
+  # architectures combo as the generated nilearn wheel is the same on all
+  # platforms (universal wheel).
+  # We run the tests on 2 different target platforms for testing purpose only.
+  # We use miniconda versions of Python provided by appveyor windows images
+  matrix:
+    - PYTHON: "C:\\Miniconda"
+      PYTHON_VERSION: "2.7.x"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Miniconda3-x64"
+      PYTHON_VERSION: "3.4.x"
+      PYTHON_ARCH: "64"
+
+install:
+  # Prepend miniconda installed Python to the PATH of this build
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+
+  # Check that we have the expected version and architecture for Python
+  - "python --version"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+
+  # Installed prebuilt dependencies from conda
+  - "conda install pip numpy=1.9.2 scipy scikit-learn nose wheel -y -q"
```

Comment:
> The nilearn installation instruction line installs it as a dependency. But I can add it there to make it more obvious.

I missed that, yes please install it with conda this is more in line with nilearn installation instructions.

The general guideline is to install everything with conda if it is available and the rest with pip.